static/sharing.js

```diff
@@ -124,7 +124,13 @@ function initShareButton(getLink, layout) {
         placement: 'bottom',
         trigger: 'manual'
     }).click(function () {
-        getLink.popover('show');
+        // load opposite state on click to prevent unnecessary popover rendering
```

Comment:
Shouldn't `.popover('toggle')` do the same, but with less boilerplate?

Reply:
Totally right, I should've done a little more research haha - commit coming right up!